plugin/core/views.py

```diff
@@ -379,8 +384,8 @@ def show_lsp_popup(view: sublime.View, contents: str, location: int = -1, md: bo
         flags=flags,
         location=location,
         wrapper_class=wrapper_class,
-        max_width=int(view.em_width() * 120.0),  # Around 120 characters per line
-        max_height=1000000,
+        max_width=_chars_to_pixels(view, userprefs().popup_max_characters_width),
+        max_height=_chars_to_pixels(view, userprefs().popup_max_characters_height),
```

Comment:
It's an approximation so not a big deal but `view.line_height()` would be more appropriate for height.